cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ACLineSegmentConversion.java

```diff
@@ -118,41 +117,33 @@ private void convertLineAtBoundary(int boundarySide) {
     // Before creating the TieLine the initial branches are reoriented if it is necessary,
     // then the setG1, setB1 and setG2, setB2 will be associated to the end1 and end2 of the reoriented branch
     private static Line createTieLine(Context context, String boundaryNode, BoundaryLine boundaryLine1, BoundaryLine boundaryLine2) {
```

Comment:
Update the comment. Now, it does not make sense to reorient the branches as in the `danglingLine `only the total shunt admittance is recorded (G and B)

Reply:
I have removed the comment.